### ISSUE TITLE
WFS200 FutureWarning

### ISSUE THREAD

**Issue by huard** [geopython/OWSLib#594]
Line https://github.com/geopython/OWSLib/blob/f48d8f69ada2a746072b16c8b3cba1b62c5fd1ec/owslib/feature/wfs200.py#L153
raises 
```
FutureWarning: The behavior of this method will change in future versions. Use specific 'len(elem)' or 'elem is not None' test instead.
```

**Comment by cehbrecht** [comment]
@huard would you like to provide a fix?

**Comment by cehbrecht** [comment]
@huard ok, already fixed by #595.
